### ISSUE TITLE
Pelias Geocoding

### ISSUE THREAD

**Issue by zabuTNT** [openmove/odh-mentor-otp#23]
- [ ] Pelias service for geocoding for south tyrol region.
- [ ] custom pois?

Resources:

1. openaddress:

- it/32/south-tyrol-de
- it/32/south-tyrol-it
- it/32/south-tyrol-lld

2. whatsonfirst:
https://spelunker.whosonfirst.org/id/85685271/

**Comment by stemove** [comment]
also this to payment servce(always pelias in background)
https://geocode.earth/

**Comment by stemove** [comment]
may be some problems is that the gtfs is outside the boundaries of Bolzano

**Comment by stemove** [comment]
this Pelias plugin is useful for import NOI Point Of Interest data:
https://github.com/pelias/csv-importer

NOI POIs must be imported as Venue layer:
https://github.com/pelias/documentation/blob/4ac70226018718e3d89b293080e48299962283ce/autocomplete.md#layers

support multilangs:
https://github.com/pelias/csv-importer#names-in-multiple-languages

**Comment by stemove** [comment]
importing too many different data into pelias can create problems in the expected results, it is better to import the minimum data used for this application, for example gtfs and NOI POIs

In my case study, whosonfirst occupies 27GB on disk(cache) and 4GB on ram(for Trento)

this is to be evaluated if it is convenient

**Comment by stemove** [comment]
this GTFS datasource can also be added through this pelias module

https://github.com/pelias/transit

using similar config(this is for Trento):
```json
"transit": {
      "datapath": "/data/transit",
      "feeds": [
        {
          "layerId": "stops",
          "url": "http://www.ttesercizio.it/opendata/google_transit_urbano_tte.zip",
          "filename": "stops.txt",
          "agencyId": "tte",
          "agencyName": "trentino trasporti esercizio",
          "layerName": "stop"
        }
      ]
    }
```

**Comment by stemove** [comment]
tool to download data WOF:
https://spelunker.whosonfirst.org/search/?q=trentino

https://spelunker.whosonfirst.org/id/404227499/

**Comment by stemove** [comment]
Pelias transit module loads followe data from gtfs datasource:
- stops
- landmarks
- intersections
- bikeshare
- fare

**Comment by stemove** [comment]
mapping fields of custom pois:
```
// 20201111010909
// https://tourism.opendatahub.bz.it/api/Accommodation?language=de&poitype=447&active=true&fields=Id,AccoDetail.de.Name,Latitude,Longitude&pagesize=100

{
  "OnlineResults": -1,
  "ResultId": "",
  "TotalResults": 9592,
  "TotalPages": 96,
  "CurrentPage": 1,
  "Seed": null,
  "Items": [
    {
      "Id": "2657B7CBCB85380B253D2FBE28AF100E",
      "AccoDetail.de.Name": "1477 Reichhalter",
      "Latitude": 46.615101,
      "Longitude": 11.143296
    },
    {
      "Id": "C93CE627A8497E697C657CF53028F9D2",
      "AccoDetail.de.Name": "360 Grad - Bundeswehr Sozialwerk",
      "Latitude": 46.660809,
      "Longitude": 11.180298
    },
```

**Comment by stemove** [comment]
feature branch work in progress here:
https://github.com/openmove/odh-mentor-otp/tree/pelias-lite

**Comment by stemove** [comment]
case study:
https://www.suedtirolmobil.info/de/

**Comment by stemove** [comment]
solved in #38
